tracing-journald/src/lib.rs

```diff
@@ -85,6 +85,7 @@ pub struct Subscriber {
     socket: UnixDatagram,
     field_prefix: Option<String>,
     syslog_identifier: String,
+    additional_fields: HashMap<String, String>,
```

Comment:
```suggestion
    additional_fields: Vec<u8>,
```
We don't need to random-access fields, so we might as well serialize it ahead of time so including it in the output is a simple memcpy.

Reply:
Done